.gitignore

```diff
@@ -14,3 +14,5 @@ out/
 .classpath
 .attach*
 src/test/resources/application-test.yml
+kind
+kubectl
```

Comment:
I added these so that we can run `.github/setup-unit.sh` locally to create a Kind cluster :)